docs/pages/database-access/guides/mongodb-atlas.mdx

```diff
@@ -225,6 +225,58 @@ certificate with `CN=alice` subject.
   Case matters so make sure to specify Common Name in the username with capital
   letters `CN=`.
 </Admonition>
+</TabItem>
+
+<TabItem label="AWS IAM authentication">
+Teleport will authenticate with MongoDB Atlas using [AWS IAM roles](https://www.mongodb.com/docs/atlas/security/passwordless-authentication/).
```

Comment:
I don't think we need to take the reader outside of this page here. Can we briefly describe how Atlas uses IAM roles for authentication, then include this link in the "Further reading" section instead of here?